src/TuDa.CIMS.Web/Components/ShoppingCart/ShoppingCartSubmitPopupWorkingGroupSelection.razor.cs

```diff
@@ -62,10 +62,19 @@ private async Task WorkingGroupSelectedInternal(WorkingGroupResponseDto item)
         await WorkingGroupChanged.InvokeAsync(item);
     }
 
-    private Task<IEnumerable<WorkingGroupResponseDto>> Search(
+    private async Task<IEnumerable<WorkingGroupResponseDto>> Search(
         string name,
         CancellationToken token
-    ) => api.GetAllAsync(name).Match(value => value, _ => []);
+    )
+    {
+        if (string.IsNullOrWhiteSpace(name))
+        {
+            return [];
+        }
+        var workingGroups = await api.GetAllAsync(name).Match(value => value, _ => []);
+        workingGroups = workingGroups.Where(wg => wg.IsDeactivated == false).ToList();
+        return workingGroups;
+    }
```

Comment:
🔧 You are removing a change from #317 here. Please merge this correctly
```suggestion
    ) => api.GetAllAsync(name).Match(value => value.Where(wg => wg.IsDeactivated == false).ToList(), _ => []);
    
```